src/sagemaker/image_uri_config/mxnet.json

```diff
@@ -816,6 +817,36 @@
                 },
                 "repository": "mxnet-inference-eia",
                 "py_versions": ["py2", "py3"]
+            },
+            "1.7.0": {
+                "registries": {
+                    "af-south-1": "626614931356",
+                    "ap-east-1": "871362719292",
+                    "ap-northeast-1": "763104351884",
+                    "ap-northeast-2": "763104351884",
+                    "ap-south-1": "763104351884",
+                    "ap-southeast-1": "763104351884",
+                    "ap-southeast-2": "763104351884",
+                    "ca-central-1": "763104351884",
+                    "cn-north-1": "727897471807",
+                    "cn-northwest-1": "727897471807",
+                    "eu-central-1": "763104351884",
+                    "eu-north-1": "763104351884",
+                    "eu-west-1": "763104351884",
+                    "eu-west-2": "763104351884",
+                    "eu-west-3": "763104351884",
+                    "eu-south-1": "692866216735",
+                    "me-south-1": "217643126080",
+                    "sa-east-1": "763104351884",
+                    "us-east-1": "763104351884",
+                    "us-east-2": "763104351884",
+                    "us-gov-west-1": "442386744353",
+                    "us-iso-east-1": "886529160074",
+                    "us-west-1": "763104351884",
+                    "us-west-2": "763104351884"
+                },
+                "repository": "mxnet-inference-eia",
+                "py_versions": ["py3"]
```

Comment:
I believe this might need to be py36 as per https://github.com/aws/deep-learning-containers/commit/cab6c4e2d11aaf27524d56732aa37921d5308895.

Reply:
Thanks for the comment. Will update

Reply:
For MXNet 1.7.0 training image, though the [image tag](https://github.com/aws/deep-learning-containers/blob/master/available_images.md#prior-general-framework-container-versions) is `py36`, in [this file](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/image_uri_config/mxnet.json#L314) it still specifies the `py_version` as `py3`. We should keep training and inference images the same, otherwise when deploying the endpoint, it errors out because the `py_version` doesn't match. So I'll leave it as `py3` to be consistent with the training image.

Reply:
Upon checking
```
docker pull 763104351884.dkr.ecr.us-west-2.amazonaws.com/mxnet-inference-eia:1.7.0-cpu-py36
docker pull 763104351884.dkr.ecr.us-west-2.amazonaws.com/mxnet-inference-eia:1.7.0-cpu-py3
```

Are both valid, so I'm okay with having py3. We add py36 in a separate PR if desired.